tests/builders/test_builders.py

```diff
@@ -140,11 +140,10 @@ def test_file_count():
     cmd.build()
 
 
-def test_regex_check():
-    """This test buildspec using status check with  'status_regex'"""
+def test_file_regex():
```

Comment:
You deleted Test_regex_check method, you suppose to add your method and not delete the exist one.